CMakeLists.txt

```diff
@@ -72,6 +72,7 @@ option(USE_DISCORD_PRESENCE "Enables Discord Rich Presence, show the current gam
 option(USE_MGBA "Enables GBA controllers emulation using libmgba" ON)
 option(ENABLE_AUTOUPDATE "Enables support for automatic updates" ON)
 option(STEAM "Creates a build for Steam" OFF)
+option(USE_RETRO_ACHIEVEMENTS "Enables integration with retroachievements.org")
```

Comment:
This is missing a default value. In fact, I think you not setting a value here means that our CI has not actually complied your code, so we don't actually know if it compiles correctly on every system. -- And if you change it now it's cached so it won't change on rebuilds. Uuh not sure how we resolve this correctly...